converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Modules/WorkerModule.cs

```diff
@@ -55,6 +56,21 @@ public void Load(IServiceCollection services)
                 .Singleton()
                 .AsSelf()
                 .AsImplementedInterfaces();
+
+            if (_configuration.GetSection("TableStore").GetSection("Enabled").Get<bool>() == true)
+            {
+                services.Add<TableStoreService>()
```

Comment:
This should probably be moved to the `AddTableStorageDataStore` extension method. You should be able to replace implementations already registered.

Reply:
I moved all of this logic into the AddTableStorageDataStore